rhino/src/main/java/org/mozilla/javascript/optimizer/Signatures.java

```diff
@@ -92,17 +94,17 @@ interface Signatures {
      * NAME:GET:{name}: Looks up a the named value from the scope. Falls back to ScriptRuntime.name.
      */
     String NAME_GET =
-            "(Lorg/mozilla/javascript/Context;"
-                    + "Lorg/mozilla/javascript/Scriptable;"
+            "(Lorg/mozilla/javascript/Scriptable;"
+                    + "Lorg/mozilla/javascript/Context;"
                     + ")Ljava/lang/Object;";
 
     /**
      * NAME:GETWITHTHIS:{name}: Looks up a name in the scope like NAME:GET, and sets "this" in the
      * "last stored scriptable." Falls back to ScriptRuntime.getNameFunctionAndThis.
      */
     String NAME_GET_THIS =
-            "(Lorg/mozilla/javascript/Context;"
-                    + "Lorg/mozilla/javascript/Scriptable;"
+            "(Lorg/mozilla/javascript/Scriptable;"
+                    + "Lorg/mozilla/javascript/Context;"
```

Comment:
maybe a comment somewhere why the order is important (at least because we usually have the context first) might be helpful to avoid changing the order back in the future

Reply:
I updated the comments. Thanks!